collections/_article/godot-xr-update-feb-2025.md

```diff
@@ -0,0 +1,117 @@
+  - The project setting is enabled by default, which causes all runtime permissions to be requested at app launch
+  - Developers can disable that behavior so that the app / game logic can itself request the permissions in a context-specific manner
+- Add export profile for Magic Leap 2 devices
+
+### Bug Fixes
+
+- Update the signal emitted by `OpenXRFbSpatialEntity.erase_from_storage()` from `_on_save_to_storage` to `_on_erase_from_storage`
+- Only add the [Android LAUNCHER category](https://developer.android.com/reference/android/content/Intent#CATEGORY_LAUNCHER) to the generated XR binary if the `package/show_in_app_library` export option is enabled
+- Fix wall, floor and ceiling collision shapes with Jolt physics
+- Add export option to enable enable / disable sharing of Meta's spatial anchors
+  - This adds the `com.oculus.permission.IMPORT_EXPORT_IOT_MAP_DATA` permission when enabled
+- Fix crash when attempting to create a spatial anchor too early
+  - Fix a crash that happens when a spatial anchor is created before the OpenXR session has begun
+- Reworked geometric algebra used by Meta body tracking extension to address root and shoulder tracking bugs
+- Remove deprecated "Contextual" boundary mode on Meta Quest
+- Fix `OpenXRFbPassthroughExtensionWrapper` from wiping out the next pointer chain for system properties
+- Fix passthrough sample color map display bug
+- Fix the issue preventing vendor options in the export preset from being updated
+
+## Release Godot XR Tools 4.4.0
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/xJKQ2ca5zVw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+This version of the XR Toolkit has been updated to contain [Godot 4.2 mesh formats](https://godotengine.org/article/godot-4-2-arrives-in-style/#critical-and-breaking-changes) and thus **requires Godot 4.2+**.
+
+You can download version 4.4.0 from [GitHub](https://github.com/GodotVR/godot-xr-tools/releases) or the [Asset library](https://godotengine.org/asset-library/asset/1698).
+You can download [the Godot XR Tools demo on itch.io](https://godot-xr.itch.io/godot-xr-tools-demo).
+
+The Godot XR Tools repository on Github contains a [Github workflow](https://github.com/GodotVR/godot-xr-tools/blob/master/.github/workflows/publish-demo-on-push.yaml) that prepares and uploads the demo project to the above itch.io page. This workflow can be used as a template for your own project.
+
+### Features
+- The StartXR startup script has had a cleanup pass, it now:
+  - properly handles the pass through system changes in Godot 4.3. Godot 4.3 saw changes in basing pass through on the environment blend mode and moved logic into the vendor plugin to improve platform support for pass through. XR Tools now makes use of this new system.
+  - adds proper support for the immersive-ar and immersive-vr webXR modes
+- provides signals to notify when the user enters or exits XR
```

Comment:
```suggestion
- Provide signals to notify when the user enters or exits XR
```

Reply:
I think this was meant to be indented one more level, to come under the "it now:"

Reply:
I wondered that, but wasn't sure if it was something a startup script would do.

Reply:
Yeah, copy pasting from google docs removes all the formating, so I missed that one